README.md

```
@@ -1,58 +1,129 @@
+  <img src="https://img.shields.io/badge/Asterisk-FF6600?style=for-the-badge&logo=asterisk&logoColor=white" alt="Asterisk" />
+  <img src="https://img.shields.io/badge/XMPP-007BFF?style=for-the-badge&logo=xmpp&logoColor=white" alt="XMPP" />
+</p>
+
+</div>
+
+---
+
+## 📊 GitHub Analytics
+
+<div align="center">
+  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=dessanhemrayev&show_icons=true&theme=tokyonight&include_all_commits=true&count_private=true&hide_border=true"/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)